src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/utils/AsylumCaseUtils.java

```diff
@@ -291,7 +291,7 @@ public static String calculateFeeDifference(String originalFeeTotal, String newF
 
             BigDecimal originalFee = new BigDecimal(String.valueOf(Double.parseDouble(originalFeeTotal) / 100));
             BigDecimal newFee = new BigDecimal(String.valueOf(Double.parseDouble(newFeeTotal) / 100));
-            BigDecimal difference = originalFee.subtract(newFee);
+            BigDecimal difference = originalFee.subtract(newFee).abs();
```

Comment:
can you add a test to prevent negative numbers?

Reply:
Thanks Miguel I added a test case for 3 scenarios, positive difference, negative difference and no difference